irrep/spacegroup.py

```diff
@@ -591,7 +589,7 @@ def _findsym(self, inPOSCAR, cell):
                 dataset['international'],
                 dataset['number'], 
                 cell[0], 
-                np.linalg.inv(dataset['transformation_matrix']),
+                dataset['transformation_matrix'],
                 dataset['origin_shift']
```

Comment:
I put this inverse inside `determine_basis_transformation`. The idea is to keep all the steps related to the manipulation of `refUC` and `shiftUC` inside that function. Like that, we'll have to look only into that function when changing this manipulation.